superset/charts/data/api.py

```diff
@@ -143,7 +143,7 @@ def get_data(self, pk: int) -> Response:
             query_context = self._create_query_context_from_form(json_body)
             command = ChartDataCommand(query_context)
             command.validate()
-        except DatasourceNotFound as error:
```

Comment:
I know some people find Pylint to be somewhat cumbersome, but I love to see Pylint every evolving which results in fixes like this—which typically `flake8` et al. miss.